pkg/cmd/roachtest/tests/cluster_to_cluster.go

```diff
@@ -404,31 +424,31 @@ func registerClusterToCluster(r registry.Registry) {
 					t.L().Printf("src cluster workload initialization took %d minutes", int(initDuration.Minutes()))
 				}
 
+				t.L().Printf("begin workload on src cluster")
+				workloadCtx, workloadCancel := context.WithCancel(ctx)
+				defer workloadCancel()
+
+				workloadDoneCh := make(chan struct{})
+				go func() {
```

Comment:
Any reason we're not the monitor here?

Reply:
it didn't seem immediately necessary, as the RunE cmd requires the workloadCtx. The main advantage of using a monitor is guaranteed clean goroutine cleanup during an unhappy shutdown, right?

Reply:
Yes, that would be the main advantage.

Reply:
changed. done.